apps/webapp/app/v3/services/triggerTask.server.ts

```diff
@@ -183,6 +187,41 @@ export class TriggerTaskService extends BaseService {
               }
             }
 
+            if (body.options?.queue) {
+              const concurrencyLimit = body.options.queue.concurrencyLimit
+                ? Math.max(0, body.options.queue.concurrencyLimit)
+                : null;
+              const taskQueue = await prisma.taskQueue.upsert({
+                where: {
+                  runtimeEnvironmentId_name: {
+                    runtimeEnvironmentId: environment.id,
+                    name: queueName,
+                  },
+                },
+                update: {
+                  concurrencyLimit,
+                  rateLimit: body.options.queue.rateLimit,
+                },
+                create: {
+                  friendlyId: generateFriendlyId("queue"),
+                  name: queueName,
+                  concurrencyLimit,
+                  runtimeEnvironmentId: environment.id,
+                  projectId: environment.projectId,
+                  rateLimit: body.options.queue.rateLimit,
+                  type: "NAMED",
+                },
+              });
+
+              if (typeof taskQueue.concurrencyLimit === "number") {
```

Comment:
What happens here if they previous had a concurrency limit set, and then removed it? These limits wouldn't get cleared. Looks like we might have the same issue here: https://github.com/triggerdotdev/trigger.dev/blob/main/apps/webapp/app/v3/services/createBackgroundWorker.server.ts#L175

Reply:
Good point, I'll take a look at that